### PR TITLE
results

### PR DESCRIPTION
**Pull request by I11ia**
- [DEMO LINK](https://I11ia.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://I11ia.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,51 @@
...
+        class="logo"
+        src="/images/logo.png"
+        alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="apple" >Apple</a>
+          </li>
+          <li class="nav__item">
```
Comment (Esceype) [inline src/index.html:29]: Add empty line between sibling blocks

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,51 @@
...
+            <a class="nav__link" href="samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            data-qa="hover"
+            href="laptops & computers">Laptops & Computers</a>
```
Comment (Esceype) [inline src/index.html:39]:
```suggestion
            href="laptops & computers"
          >
            Laptops & Computers
          </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+.nav__link {
+  display: block;
+  padding: 23px 0 ;
+  font-size: 12px;
+  margin-left: 20px;
+  text-decoration: none;
+  color: #000;
+}
+
+a:hover {
```
Comment (Esceype) [inline src/style.css:49]: Use class selector

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
```
Comment (Esceype) [inline src/style.css:7]: Better to set it only in that selector where you need it

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ!!! Read the following comments bellow

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,89 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com">
```
Comment (denys-danyliuk) [inline src/index.html:13]:
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com">
```

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,89 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
```
Comment (denys-danyliuk) [inline src/index.html:17]: it should be in the middle, have you ever seen header like this?
Знімок екрана 2023-08-08 о 10 19 46

File: src/index.html [hunk 7]
```diff
@@ -7,8 +7,89 @@
...
+      <a href="index.html">
+        <img
+        class="logo"
+        src="/images/logo.png"
+        alt="moyo logo">
```
Comment (denys-danyliuk) [inline src/index.html:22]: 13'th point in checklist

File: src/index.html [hunk 8]
```diff
@@ -7,8 +7,89 @@
...
+            data-qa="hover"
+            href="laptops & computers"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+            href="gadgets"
```
Comment (denys-danyliuk) [inline src/index.html:62]: 13'th point in checklist

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,68 @@
...
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:26]: 3'rd point in checklist

File: src/index.html [hunk 10]
```diff
@@ -7,8 +7,89 @@
...
+        <img
+        class="logo"
+        src="/images/logo.png"
+        alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active"
+            href="apple"
```
Comment (denys-danyliuk) [inline src/index.html:28]:
```suggestion
            href="#apple"
```

Comment (denys-danyliuk) [reply src/index.html:28]: add anchors if there is no better option

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  max-width: 1200px;
```
Comment (denys-danyliuk) [inline src/style.css:7]: don't hardcode it like this

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,68 @@
...
+  height: 60px;
```
Comment (denys-danyliuk) [inline src/style.css:8]: Знімок екрана 2023-08-08 о 10 22 42

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -7,8 +7,74 @@
...
+
+          <li class="nav__item">
+            <a class="nav__link" href="smartphones">
+            Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+            class="nav__link"
```
Comment (anastasiiavorobiova) [inline src/index.html:45]:
```suggestion
              class="nav__link"
```
Consider fixing indentations accordingly to the code style guide kit

File: src/index.html [hunk 14]
```diff
@@ -7,8 +7,74 @@
...
+            data-qa="hover"
```
Comment (anastasiiavorobiova) [inline src/index.html:46]:
```suggestion
              data-qa="hover"
```

File: src/index.html [hunk 15]
```diff
@@ -7,8 +7,74 @@
...
+            href="#laptops & computers"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+            Gadgets
```
Comment (anastasiiavorobiova) [inline src/index.html:55]:
```suggestion
              Gadgets
```

File: src/index.html [hunk 16]
```diff
@@ -7,8 +7,74 @@
...
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+            Tablets
```
Comment (anastasiiavorobiova) [inline src/index.html:61]:
```suggestion
              Tablets
```

File: src/index.html [hunk 17]
```diff
@@ -7,8 +7,74 @@
...
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+            Photo
```
Comment (anastasiiavorobiova) [inline src/index.html:67]:
```suggestion
              Photo
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Fantastic job! Almost done!

File: src/style.css [hunk 18]
```diff
@@ -1,3 +1,71 @@
+html {
+  --active-color: #00abcd;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:7]: 3'rd point in checklist, I wrote you about it before

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by Esceype — APPROVED** [review]
Great job!!!

**Pull request closed without merging**